File: Kylie_Guzman/henry_guzman/mis_programas/dart/repasando02.dart
import 'dart:io';

void main() {

  int personas = 0;
  int num = 0;

  while(personas < 101 ){

    print("has ingresado a un concurso");

    print("ingresa tu nombre");
    String name = stdin.readLineSync()!;

    num = num + 1;
    personas = personas + 1;

    if(num == 28){

      print("$name has ganado!!");

    }else(){
          
        print("$name lamentablemente has perdido");
    };
    
  }
}

File: Kylie_Guzman/henry_guzman/mis_programas/ruby/repasando02.rb

personas = 0;
num = 0;

while personas < 101

  puts "has ingresado a un concurso"

  puts "ingresa tu nombre"
  name = gets.chomp

  num = num + 1
  personas = personas + 1
    
  if num == 28

    puts "#{name} has ganado!!"

  else
          
    puts "#{name} lamentablemente has perdido"
  end
    
end


